src/app/collection/shared/included/feature/feature.component.html

```diff
@@ -5,7 +5,7 @@
              <h2 class="featured-learning-title">
                  FEATURED LEARNING OBJECTS
              </h2>
-             <p>
+             <p *ngIf="collectionName !== 'nice' else niceDescription">
```

Comment:
Why was this added?

Reply:
This component was in shared, so I thought it was being used in other places, so I made it so that the changes in this component are just to the nice collection. I know that it's actually only being used for the nice collection, would it be better to remove the conditional?